components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/LiteServiceProvider.java

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.application.common.model;
+
+/**
+ * This class represents the basic service provider information.
+ * The purpose of this class is to store load service provider configuration with minimal load.
+ */
+public class LiteServiceProvider {
```

Comment:
Would be good to have a better name..
Lite might give a wrong idea about this class :)

Reply:
(This not actually the lite version of a service provider but a model we are using the extract a part of the application)

Reply:
ServiceProviderMetadata?